doc/index.md

```diff
@@ -0,0 +1,338 @@
+
+ ```csharp
+ // Get portal item
+var portal = await ArcGISPortal.CreateAsync().ConfigureAwait(false);
+var item = await PortalItem.CreateAsync(portal, AppSettings.CurrentSettings.PortalItemID).ConfigureAwait(false);
+
+// Test if mmpk is not already downloaded or is older than current portal version
+if (!this.Files.Contains(this.TargetFileName) ||
+    item.Modified.LocalDateTime > AppSettings.CurrentSettings.MmpkDownloadDate)
+{
+    this.IsDownloading = true;
+    this.DownloadURL = item.Url.AbsoluteUri + "/data";
+}
+else
+{
+    this.IsReady = true;
+}
+ ```
+
+![App Start Check](./images/appstartcheck.png)
+
+If a download is needed, the app uses the iOS [NSUrlSessionDownloadTask](https://github.com/xamarin/ios-samples/blob/master/SimpleBackgroundTransfer/SimpleBackgroundTransfer/SimpleBackgroundTransferViewController.cs) to download the mmpk. This insures that the mmpk is downloaded even if the user switches away from the app during the download.
+
+```csharp
+// Create a new download task.
+var downloadTask = this.session.CreateDownloadTask(NSUrl.FromString(downloadUrl));
+```
+
+Both the mmpk and the app settings files are stored on the device and once downloaded, the app can work in a fully disconnected environment.
+
+### Loading the map
+
+![Initial Extent](./images/initialextent.png)
+
+Once everything is downloaded, the mmpk and all of its contents are loaded inside the Initialize method of the Main View Model.
+
+```csharp
+internal async Task InitializeAsync()
+{
+    // Load mmpk from device and load it
+    var mmpk = await this.LoadMMPKAsync().ConfigureAwait(false);
+
+    // Display map from the mmpk. Assumption is made that the first map of the mmpk is the one used
+    this.Map = mmpk.Maps.FirstOrDefault();
+    await Map.LoadAsync().ConfigureAwait(false);
+
+    // Set the locator to be used in the app
+    var locator = mmpk.LocatorTask;
+    await locator.LoadAsync().ConfigureAwait(false);
+
+    // Create instance of the Location View Model
+    if (LocationViewModel.Instance == null)
+    {
+        LocationViewModel.Instance = LocationViewModel.Create(Map, locator);
+    }
+
+    // Set viewpoint of the map depending on user's setting
+    await this.SetInitialViewPointAsync().ConfigureAwait(false);
+}
+```
+
+The Initial Viewpoint is the initial extent that the map is loaded to when the application starts. This extent is configured in the settings. If the initial extent is not set, the map will display the full extent of the mmpk.
+
+### Navigating the map
+
+There are three ways to move around the map: pan, zoom and floor change. The floor picker is displayed only when the map is zoomed in past a certain extent. This extent is also defined in the settings and can be easily changed. When the floor picker is activated, the app applies [definition expressions](/net/latest/ios/api-reference/) on the rooms and walls layers to only display the selected floor.
```

Comment:
`[definition expressions](/net/latest/ios/api-reference/)`

Relative path needs the https://developers.arcgis.com before it